File: baekjoon/10807_개수세기.py
import sys
sys.stdin = open('10807.txt', 'r')

N = int(input())
arr = list(map(int, input().split()))
find = int(input())

cnt = 0
for i in range(N):
    if arr[i] == find:
        cnt += 1

print(cnt)

File: baekjoon/10807_개수세기.cpp
#include <iostream>
using namespace std;

int main() {
	ios::sync_with_stdio(0);
	cin.tie(0);
	int N;
	cin >> N;
	int arr[100];
	for (int i = 0; i < N; i++) {
		cin >> arr[i];
	}
	int find;
	cin >> find;
	int cnt = 0;
	for (int i = 0; i < N; i++) {
		if (find == arr[i]) {
			cnt++;
		}
	}

	cout << cnt << "\n";

	return 0;
}